### DISCUSSION TITLE
2024-08-30 - 🧠⚡Sydney Brainstorming 2024 - Retro - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by bradystroud** [SSWConsulting/SSW.Rules.Content#9074] [Chewing the Fat]
AI Account Manager (Lev.AI) | Sydney Brainstorming 2024
https://youtu.be/Z9uj-yhXbZs (25 min)

What are your thoughts on this idea?

**Comment by tiagov8** [comment]
I've seen this idea (of facilitating client booking) in other forms over the years, but this AI method is certainly more promising than what we tried before.

My suggestion is to call it **SSW Bridge**

I'm curious, is this idea something that:

**A.** Addresses a specific need or problem we've identified with clients being blocked
or 
**B.** is more inspired by SSW's vision or desire to innovate

If A, I think this is super important. If B, I would give it a lower priority.

> "Necessity is the mother of invention."

**Comment by TobyChurches** [comment]
> What are your thoughts on this idea?

I really like the Idea and Calum's POC was great! There is a bit of risk but with some testing I think this could be a very valuable tool for SSW

**Comment by tkapa** [comment]
Seems ok - I'd only support it if account managers think that this could save time...

**Comment by jakebayliss** [comment]
If its close to being done already I think we should get it to a working state, get the acc managers to use it for a week or 2 and see if it actually helps them save time. If it does then carry on, if not then leave it

**Comment by GordonBeeming** [comment]
I think it looks promising
Even if it just provides the account managers with all the context and information needed it would be a huge help
I wouldn't want customers directly integrating with it
My only concern is that we have loads of promising ai balls in the air right now

**Comment by ozdemir-mehmet** [comment]
It was hard to watch 8 videos in 15 mins but I gave it a crack.

**Reply by wicksipedia** [reply]
watch in parallel

**Reply by jakebayliss** [reply]
The old mehmet wouldve watched them all in 0.5 speed

**Comment by StefStar** [comment]
I think Lev.AI is a great idea and will work well for future but i'd be more interested in driving a lot more leads to the account managers first before we make their life too easy :)

**Comment by ricksu978** [comment]
Agree we should start using it and get feedback => improve it => use it => get feedback => improve it => use it => get feedback (loopppping)

**Comment by Calinator444** [comment]
> AI Account Manager (Lev.AI) | Sydney Brainstorming 2024
https://youtu.be/Z9uj-yhXbZs (25 min)
What are your thoughts on this idea?

I mentioned this on the GitHub issue, but I don't think the average client will trust AI to contact the right people or make bookings for them based on what is essentially a free form text input. If we do go ahead with this idea we need to be transparent about how it works, i.e. In the response email you could mention that a Sysadmin is reviewing their ticket. Then you could reply when the booking has been sent.

On the other hand it looks like the prototype for Lev.AI is pretty much ready to go, so if it requires minimal development time to polish I'd just roll it out still include the option to talk to an account manager in person.

**Comment by isaaclombardssw** [comment]
It feels more novelty than a practical utility currently – interfacing with a real person makes a big difference in building trust initially.

Levi suggested, in the talk, of using it as a segue to talk with clients about AI, applications of AI etc. and I think for that it's worth trialing. If it leads to some extra efficiency that's a bonus.

**Comment by chrisschultzssw** [comment]
It's a great idea - let's get it fully working, and get some metrics on how much time it saves our account managers.

**Comment by Marxoz** [comment]
Even though Levi.AI is a good idea, I don't see it working in SSW. But would love to give it a go...

**Comment by RobJT-wq** [comment]
I think it's a good idea and it looks like we'd be able to start using the solution soon.

**Comment by wicksipedia** [comment]
> What are your thoughts on this idea?

we could just have an online store on the website and customers book+prepay people/experts 🤷‍♂️

immediate feedback for the person booking
less AI = less CO2

**Comment by kikibianc** [comment]
Idea is amazing and I think we should expand on it and implement it, then gather data and feedback and see if it's worth to keep using.
1. Clients might not trust the AI to book or give good answers.
2. Even if clients don't trust the AI, account managers could just use it to make their jobs easier - it is tiring to go and book 50 devs in different clients and places, if AI could do it with a prompt that's more time to focus on clients
3. Having AI connect to the different SSW's databases (CRM, Zendesk, SQL, TimePro, Sugarlearning, Rules...) in and on itself is golden - imagine you could ask "SSW Big Brother" AI anything that's happening at SSW and get insights on it.
- Hey AI, how's SSW's forecast for this year?
- Hey AI, how many rules talk about emails? How many sugarlearning items? Are they linked?
- Hey AI, who's in Brisbane today? Who are they working for? Did they send their daily scrum? Have they checked any emails this week? Who's on a mac and who's on a pc?
- Hey AI, where is Adam right now? Is he attending any events? Is SSW hosting any events?

**Comment by bettybondoc** [comment]
🚀 Ship it!

If it works out well, then we can help our account managers + have a potentially sellable product in our hands.
If not, we can still showcase this as an AI project we could demo during conferences or when clients are looking for sample AI works from SSW.

**Comment by MarcusIrmscherSSW** [comment]
Its a good idea, but the name could do with some workshopping. 💡

**Comment by Ben0189** [comment]
I just tested it and see this as a potentially great solution. The estimated two-week timeframe is reasonable and worth trying👍.

Recommendation: Consider renaming it to something that sounds like AI without explicitly using the term "AI."

**Comment by awaltos** [comment]
Great presentation - very well set out, very entertaining and having a working POC is awesome. 
I don't think there is much value to this as a product if the manager needs to verify/double check things anyway. Seems like just another source to receive client queries from. Perhaps it could serve as a demo for clients who may want this kind of thing implemented for their own use case but seeing as it only took a day to stand up a POC and the idea is not that abstract it could probably be just talked about in a client meeting or spec review.

**Comment by ncn-ssw** [comment]
Why tie this to emails, rather than providing a convenient UI that can be queried?

**Comment by miichaelsmedley** [comment]
> AI Account Manager (Lev.AI) | Sydney Brainstorming 2024 https://youtu.be/Z9uj-yhXbZs (25 min)
> 
> What are your thoughts on this idea?

like it - should be part of a capability statement rather than a product

**Comment by amankumarrr** [comment]
I like the idea. Maybe we could add '(Check by xxx)' at the top, indicating the reviewer who will approve the prompt, to incorporate a human touch alongside the AI response.

**Comment by bahjatalaadelssw** [comment]
Let's just decide if SSW wants to transition into a product company 🤔

**Comment by jimmidier** [comment]
Cool idea and great demo! The name could be more related to the company brand though.

**Comment by Tanya-Leahy** [comment]
Great idea and I can see this being a sellable product.

**Comment by jerryluossw** [comment]
Sounds good! If the account manager finds it useful, we can expand it to other types of emails.

**Comment by Aibono1225** [comment]
this looks really cool! I like the POC which seems very promising

**Comment by tino-liu** [comment]
Cool product demo! I do think this could scale beyond emails.

**Comment by AnkyChe** [comment]
Great presentation! I am looking forward to seeing it becoming a sellable product.

**Comment by babakamyljanovssw** [comment]
Good idea and presentation! Agree with others that the name needs to be related to brand

**Comment by Freego1783** [comment]
Looks promising! I don't know enough about the daily struggles of account managers, but if they think it would help then it seems reasonable to give it a go.

**Comment by matt-goldman-ssw** [comment]
Cool idea, cool demo, great presentation. Could be a valuable product to sell, not convinced it's what we need at SSW.

**Comment by drwharris** [comment]
Would be a nice tool to demo but cant see it being used overly much

**Comment by MattParkerDev** [comment]
An interesting idea, however if an account manager needs to go and double check that the information in the proposed email is correct, it feels like they are doing the same amount of work.

**Comment by vladislav-kir** [comment]
> What are your thoughts on this idea?

This sounds like a cool idea, but how much time will it actually save, because I assume the account managers will still have to manually check that AI did not mess anything up...

**Comment by PennyWalker** [comment]
Love the idea! We are so busy and anything that saves us time in the long run would be awesome!
